### PR TITLE
Add e2e for create tools

### PR DESCRIPTION
**Pull request by apajaro**

### CODE REVIEW

File: src/Tools/CreateTools/CreateConeTool.js [hunk 1]
```diff
@@ -57,13 +59,20 @@ class CreateConeTool extends CreateGeomTool {
...
    */
   createRelease(pt) {
-    if (this._radius == 0 || this._height == 0) {
+    if (this._radius == 0) {
       UndoRedoManager.getInstance().undo(false)
       this.stage = 0
       this.emit('actionFinished')
     }
     if (this.stage == 1) {
+      console.log('Going in boy')
```
Comment (phtaylor) [inline src/Tools/CreateTools/CreateConeTool.js:68]: oops

**Review by phtaylor — APPROVED** [review]
just one minor issue.

**Comment by phtaylor** [comment]
Maybe the visual changes need to be approved.

File: src/Tools/CreateTools/CreateConeTool.js (hunk 1 above)
Comment (apajaro) [reply src/Tools/CreateTools/CreateConeTool.js:68]: lol

**Pull request merged**